Tests/RxCocoaTests/Control+RxTests+UIKit.swift

```diff
@@ -616,10 +616,20 @@ extension ControlTests {
 
 // UISearchBar
 extension ControlTests {
-    func testSearchBar_DelegateEventCompletesOnDealloc() {
-        let createView: () -> UISearchBar = { UISearchBar(frame: CGRectMake(0, 0, 1, 1)) }
-        ensurePropertyDeallocated(createView, "a") { (view: UISearchBar) in view.rx_text }
-    }
+  func testSearchBar_DelegateEventTextCompletesOnDealloc() {
+      let createView: () -> UISearchBar = { UISearchBar(frame: CGRectMake(0, 0, 1, 1)) }
+      ensurePropertyDeallocated(createView, "a") { (view: UISearchBar) in view.rx_text }
+  }
+  
+  func testSearchBar_DelegateEventSearchCompletesOnDealloc() {
+    let createView: () -> UISearchBar = { UISearchBar(frame: CGRectMake(0, 0, 1, 1)) }
+    ensureEventDeallocated(createView) { (view: UISearchBar) in view.rx_searchTapped }
+  }
+  
+  func testSearchBar_DelegateEventCancelCompletesOnDealloc() {
+    let createView: () -> UISearchBar = { UISearchBar(frame: CGRectMake(0, 0, 1, 1)) }
+    ensureEventDeallocated(createView) { (view: UISearchBar) in view.rx_cancelTapped }
```

Comment:
We would also want to add unit tests for 2 new extensions to verify that calling delegate methods works. Take a look at the tests we've added to `UITableViewDelegate` wrappers and `testTableView_ModelSelected_rx_itemsWithCellFactory`.

The reason why those tests don't exist for text property is because it's not trivial to trigger text changed on ui element in unit tests.